src/commands/add.rs

```diff
@@ -33,7 +34,7 @@ pub fn add_file(args: &[String])
     let (entry, index) = image.create_file_entry(fat_file_name)?;
 
     // Get free FAT entries, fill sectors with file data.
-    for chunk in &file.bytes().chunks(fat::BYTES_PER_SECTOR) {
+    for chunk in &file.bytes().chunks(bpb.bytes_per_sector as usize) {
```

Comment:
Might as well use image.sector_size(), unless you expect to need other bios params in this function.

Reply:
I feel like there was a reason I went that route... but it's long-forgotten now :-)